frontend/src/model/releases/ReleaseDisplay.tsx

```diff
@@ -22,10 +22,12 @@ export default function ReleaseDisplay({
   model,
   release,
   latestRelease,
+  displayReviewBanner = true,
```

Comment:
Usually optional boolean props default to false. Consider reversing the logic and renaming to `hideReviewBanner`. That way when hiding the banner you just need to pass in `hideReviewBanner` instead of `displayReviewBanner={false}`.